languages/rusin/apk/build.gradle

```diff
@@ -0,0 +1,8 @@
+//owner unknown (may have not been published yet)
+ext.shouldBePublished = false
+
+apply from: "$rootDir/gradle/pack_apk.gradle"
+
+dependencies {
+    implementation project(path: ':languages:rusin:pack')
```

Comment:
So, to be clear, this pack should be separate from Serbian?

Reply:
also, Wikipedia says it should be `Rusyn`: https://en.wikipedia.org/wiki/Rusyn_language
Is that the English spelling and `rusin` is the native spelling?

Reply:
Language is Rusyn, spelled in English. Should it be called in English?

Reply:
Country of origin is Serbia but language is different that Serbian.

Reply:
it is fixed to be Rusyn everywhere